graphql_compiler/__init__.py

```diff
@@ -132,3 +141,36 @@ def graphql_to_gremlin(schema, graphql_query, parameters, type_equivalence_hints
         schema, graphql_query, type_equivalence_hints=type_equivalence_hints)
     return compilation_result._replace(
         query=insert_arguments_into_query(compilation_result, parameters))
+
+
+def get_graphql_schema_from_orientdb_records(schema_records, class_to_field_type_overrides=None,
+                                             hidden_classes=None):
+    """Construct a GraphQL schema from a list of OrientDB schema records.
+
+    Args:
+        schema_records: list of pyorient.otypes.OrientRecord describing the OrientDB schema.
```

Comment:
Let's not take in as input objects from a library that we don't depend on. Why not make the caller do the work of converting from whichever type their database client returns to something common like `dict`?